allensdk/brain_observatory/behavior/schemas.py

```diff
@@ -24,10 +24,10 @@ class SubjectMetadataSchema(RaisingSchema):
     neurodata_doc = "Metadata for an AIBS behavior or behavior + ophys subject"
     # Fields to skip converting to extension
     # In this case they already exist in the 'Subject' builtin pyNWB class
-    neurodata_skip = {"age", "genotype", "sex", "subject_id"}
+    neurodata_skip = {"age_in_days", "genotype", "sex", "subject_id"}
 
-    age = fields.String(
-        doc='Age of the specimen donor/subject',
+    age_in_days = fields.String(
```

Comment:
Is it the case that `age` data we input is always in days? I recall seeing some input JSON files where the `age` would be like `4 W` (presumably weeks?)

Reply:
We identified these in the current data set and adjusted them to the 'days' version of the ages.  The "weeks" entries are kind of a dumb LIMS relic. I'm okay with this

Reply:
@wbwakeman I'm think there are other ways of representing age that I've seen (e.g. `P1`, `P2`, `P32`) where the `P` represents `postpartum`. Have those also been converted as well? or are we okay with a mix of `ints` and `strings`?

Reply:
I'm not totally sure what your question is.  This project will have age values from P120 to P236.  You are correct that LIMS also has embryonic entries such as 'E13.5'.  I think most mouse age.name entries in LIMS have the preceding letter. I don't thing there are ever 'ints', are there?

Reply:
@wbwakeman @njmei I'm not a big fan of the rename to `age_in_days` for the reason that `ages.name` can have non-day values. There is a field in LIMS `ages.days` which is always numeric and I think is what we want to use for `age_in_days` but this would require a change to the LIMS strategy, etc. Marina was ok with this rename even given the type of values it has.

Reply:
Since this Is setting the value that will be written into the NWB file, Ben in the  PyNWB group had a request that I had forgotten until now that age be written into NWB using an ISO standard that would make it like 'P123D'.  I can think of a few ways to do this "right".  

1. Change the age.name values in LIMS to meet this standard (I worry that other code may be expecting what is there now)  
2. Change the LIMS strategy to return the age.days value instead of age.name; return the string age ('P123D') and  the int (123) for age_in_days. (Nervous about changing a LIMS strategy this late in the game)

Other option 3: just leave the column name as 'age' and return what is in LIMS ('P123')

I could probably be convinced to pursue option 2. Thoughts?

Reply:
As discussed in standup, because the `age` values that `Extractor` classes will be getting from LIMS/JSON are expected to have a regular pattern: f"P{age_in_days}", we can skip any LIMS strategy changes by instead applying some formatting to `age` strings obtained from `Extractors` in the `BehaviorDataTransforms` class.

Reply:
The resolution works for me